app/configurator/components/filters.tsx

```diff
@@ -1004,7 +1004,11 @@ export const TimeFilter = (props: TimeFilterProps) => {
 
     return (
       <Box>
-        {!disableInteractiveFilters && <InteractiveTimeRangeToggle />}
+        {!disableInteractiveFilters && (
+          <div style={{ marginBottom: "12px" }}>
```

Comment:
Nit: Seems like 12px should at some point be part of the theme spacing values. Not in this PR, I guess it would change too much.